docs/documentation/fr/get-started/TS for OOPers.md

```diff
@@ -0,0 +1,194 @@
+
+Il est important de se rappeler que TypeScript utilise le même _runtime_ que le JavaScript, donc toute ressource sur comment accomplir un comportement particulier (conversion d'un string à un nombre, affichage d'une alerte, écriture d'un fichier, etc.) s'appliquera également sur TypeScript.
+Ne vous limitez pas à ce qui est spécifique à TypeScript !
+
+## Repenser les classes
+
+Le C# et le Java sont ce qu'on pourrait appeler des langages à _POO obligatoire_.
+Dans ces langages, la _classe_ est l'unité basique d'organisation de code, ainsi que le conteneur de base de toutes les données _et_ de la logique à l'exécution.
+Obliger toute la fonctionnalité et toutes les données à être contenues dans des classes peut être un bon modèle de domaine pour certains problèmes, mais pas tous les domaines _doivent_ être représentés de cette façon.
+
+### Fonctions et données libres
+
+En JavaScript, les fonctions peuvent être placées n'importe où, et les données peuvent être librement relayées sans être cantonnées à une `class` ou une `struct`.
+Cette flexibilité est extrêmement puissante.
+Ces fonctions "libres" (celles qui ne sont pas associées à une classe), qui travaillent sur des données sans hiérarchie orientée objet, ont tendance à être le modèle préféré pour écrire un programme en JavaScript.
+
+### Classes statiques
+
+De plus, certaines structures du C# et du Java, comme les singletons et les classes statiques, ne sont pas nécessaires en TypeScript.
+
+## POO en TypeScript
+
+Cela dit, vous pouvez quand même utiliser des classes si vous le souhaitez !
+Certains problèmes ont une solution qui correspond à l'approche POO classique, et la capacité de TypeScript à supporter les classes JavaScript rendront ces solutions encore plus puissantes.
+TypeScript supporte plusieurs patterns comme l'implémentation d'interfaces, l'héritage, et les méthodes statiques.
+
+Nous couvrirons les classes plus tard dans ce guide.
+
+## Repenser les types
+
+La façon dont TypeScript comprend la notion de _type_ est très différente du C# et du Java.
+Explorons certaines différences.
+
+### Systèmes de Types Nominaux Réifiés
+
+En C# ou Java, toute valeur a exactement un type - `null`, une primitive, ou un type de classe.
+Nous pouvons appeler des méthodes comme `value.GetType()` ou `value.getClass()` pour obtenir le type exact à l'exécution.
+La définition de ce type réside dans une certaine classe sous un certain nom, et nous n'avons pas le droit d'utiliser deux classes à formes similaires de façon interchangeable, sauf si une relation explicite d'héritage existe ou une interface en commun est implémentée.
+
+Ces aspects décrivent un système de types _réifié et nominal_.
+Les types qu'on écrit sont présents à l'exécution, et ils sont liés à eux via leurs déclarations, pas leurs structures.
+
+### Types as Sets
+
+En C# ou Java, cela fait sens de faire une correspondance entre un type à l'exécution et sa déclarations à la compilation.
```

Comment:
"cela fait sens de [...]" vient de l'anglais "to make sense".
En français, on dira plutôt "avoir du sens".